doc/sphinx/addendum/sprop.rst

```diff
@@ -253,3 +253,8 @@ ill-typed terms to the kernel, which will fail with anomalies at `Qed`.
   fixed. In order to allow the user to work around such bugs, we leave the
   ability to unset the ``bad-relevance`` warning for the time being, so that the
   kernel will silently repair the proof term instead of failing.
+
+.. flag:: Printing Relevance Marks
+
+   This :term:`flag` enables debug printing of relevance marks. It is off by default.
```

Comment:
```suggestion
   Enables debug printing of relevance marks. It is off by default.
```

A couple thoughts:
- Can you add a definition of "relevance marks" somewhere?  Use `gdef` to include it in the glossary and if needed, `term` to reference it.  These constructs are described in `doc/sphinx/README.rst`.
- How do you turn on debug printing?
- What does a relevance mark look like?  Maybe give a very short example with `coqtop`.  Extra credit for showing both with debug printing on and off.

Reply:
>How do you turn on debug printing?

Set Printing Relevance Marks

will handle the rest later

Reply:
To use a language that users are used to, it could be `Printing SProp Marks`, or `Printing SProp Relevance`, ...

(Explaining in the manual what a relevance mark is remains also an alternative).

Reply:
changed